clicommand/annotate.go

```diff
@@ -94,6 +95,11 @@ var AnnotateCommand = cli.Command{
 			Usage:  "Append to the body of an existing annotation",
 			EnvVar: "BUILDKITE_ANNOTATION_APPEND",
 		},
+		cli.IntFlag{
+			Name:   "priority",
+			Usage:  "Priority of the annotation (1 to 10), default is 3",
```

Comment:
minor/non-blocking: do we want to extend this briefly to say what 1 means verses 10?

i.e. "Priority of the annotation (1 to 10), '1' is the highest importance. Default is 3"

Reply:
Same

Reply:
```suggestion
			Usage:  "Priority of the annotation (1 to 10). By default annotations have a priority of 3. Annotations with a priority of 10 will be shown first, and priority 1 will be shown last.",
```